benchflow-dsl/src/main/scala/cloud/benchflow/dsl/definition/configuration/BenchFlowExperimentConfigurationYamlProtocol.scala

```diff
@@ -16,6 +16,10 @@ import scala.util.Try
  */
 object BenchFlowExperimentConfigurationYamlProtocol extends DefaultYamlProtocol {
 
+  // default value since deserializing from a test definition it might not be defined
+  // before use it will be overridden
+  val defaultUsers = 0
```

Comment:
The experiment is generated from the test definition through parsing. During this process values not part of the experiment are removed. At a later stage the configuration from the exploration space are used to change the generated experiment to adjust it accordingly. Since in this process the test might not have defined the `users` we need to provide a default value, that is then overridden.